pdns/dnsdist-lua.cc

```diff
@@ -1691,6 +1691,12 @@ void setupLuaConfig(bool client)
       if (vars->count("serverTokens")) {
         frontend->d_serverTokens = boost::get<const string>((*vars)["serverTokens"]);
       }
+      if (vars->count("customResponseHeaders")) {
+        for (auto const& headerMap : boost::get<std::map<std::string,std::string>>((*vars)["customResponseHeaders"])) {
+          std::pair<std::string,std::string> headerResponse = std::make_pair(headerMap.first, headerMap.second);
```

Comment:
Is this a silly conversion?  It feels silly, but I see pair getting used a lot.

Reply:
It's not, we want to be able to iterate quickly over the headers so a vector of pairs makes sense to me.

Reply:
I missed your question on IRC yesterday but we can iterate more quickly over a vector of pairs than a set or a map because the memory is contiguous. I'm not worried about duplicated entries, since you use a map in the configuration step, and even otherwise it would be up to the admin to be careful.